cscs-checks/system/slurm/slurm_check.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+# TODO: sinfo, check if the normal, long, debug, etc... partitions are present?
+# TODO: scontrol, do we want to scontrol something specific?
+@rfm.parameterized_test(['squeue'],
+                        ['sacct'],
+                        ['sinfo'],
+                        ['scontrol'])
```

Comment:
I think we should pick the commands that actually make a difference. For example, follow a different workflow and may expose different problems. I am not sure if all of them are different. @victorusu ?

Reply:
The one that cause most amount of issues in our system is `squeue`, by far. If we are not testing all of them, I would keep with the `sacct` and `squeue`. Because `sacct` forces a RPC to the database and `squeue` can test the SLURM overall response.

Reply:
Done, scontrol and sinfo deleted